api/applicationserver_web.proto

```diff
@@ -138,7 +158,9 @@ message ApplicationWebhook {
   Message location_solved = 14;
   Message service_data = 18;
 
-  // next: 20
+  ApplicationWebhookHealth health_status = 20;
```

Comment:
Maybe mention that this is read-only? Or can users also use this to clear the health status?

Reply:
That is indeed the intent - the Console (soon) will have a button that nulls this in order to reset the health status.